src/makeRobot.js

```diff
@@ -38,7 +38,67 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    x: 0,
+    y: 0,
```

Comment:
store the coordinates in the object `coords`
```suggestion
    coords = {
      x: 0,
      y: 0,
    },
```